sdk/subscription/arm-subscriptions/package.json

```diff
@@ -2,11 +2,11 @@
   "name": "@azure/arm-subscriptions",
   "author": "Microsoft Corporation",
   "description": "SubscriptionClient Library with typescript type definitions for node.js and browser.",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Because this new release of the library is using ms-rest-js 2.0.x and ms-rest-nodeauth 3.0, it is immediately a major version bump to 2.0.0.